cfmtoolbox/plugins/uvl_import.py

```diff
@@ -397,6 +397,6 @@ def import_uvl(data: bytes):
     parser.featureModel()  # start parsing
 
     return CFM(
-        root=imported_features[0],
```

Comment:
How can the tests pass if the root element is not the last instead of the first element? :O

Reply:
The root element is the last